docs/zkapps/how-to-deploy-a-zkapp.mdx

```diff
@@ -27,19 +27,17 @@ When prompted, specify a name (can be anything), URL to deploy to, and fee (in
 MINA) to be used when sending your deploy transaction. The URL is the Mina GraphQL API
 that receives your deploy transaction and broadcasts it to the Mina network.
 Note that this URL is significant because it also determines which network you're
-be deploying to (e.g. QANet, Testnet, Mainnet, etc).
+be deploying to (for example, QANet, Testnet, or Mainnet).
 
-For Berkeley Testnet, use the following values:
+To deploy a zkApp to the Berkeley Testnet, use the following values:
 
 - **Name**: `berkeley`
 - **URL**: `https://proxy.berkeley.minaexplorer.com/graphql` or `https://berkeley.minascan.io/graphql`
 - **Fee**: `0.1`
 
 :::tip
-  If your project contains multiple smart contracts (e.g. `Foo` and `Bar`) that
-  you intend to deploy to the same network, a best practice is to follow a naming
-  pattern such as `berkeley-foo` and `berkeley-bar` when naming your deploy
-  aliases. You can change these alias names at anytime within `config.json`.
+If your project contains multiple smart contracts (for example, `Foo` and `Bar`) that
+you intend to deploy to the same network, use a naming pattern such as `berkeley-foo` and `berkeley-bar` when naming your deploy aliases. You can change these alias names at anytime in `config.json`.
```

Comment:
My only reservation in telling them to do this is that readers won't know if it's required for some reason (like code magic that depends upon it), as opposed to being any value they desire and we're just recommending a certain pattern as best practice. So, for avoidance of doubt, I'd still consider phrasing that includes best practice, suggestion, or similar.